python/py-aws-sam-translator/Portfile

```diff
@@ -34,18 +37,16 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-jsonschema \
                     port:py${python.version}-six
 
-    # tests don't really work at the moment
-    # https://github.com/awslabs/serverless-application-model/issues/991
     depends_test-append \
-                    port:py${python.version}-jsonschema \
                     port:py${python.version}-mock \
                     port:py${python.version}-parameterized \
                     port:py${python.version}-pytest \
-                    port:py${python.version}-py \
-                    port:py${python.version}-requests \
-                    port:py${python.version}-urllib3
+                    port:py${python.version}-pytest-cov \
+                    port:py${python.version}-yaml
 
     test.run        yes
+    test.cmd        py.test-${python.branch}
+    test.target     -v tests
```

Comment:
I'm not sure what's better, but `-v` could probably also be put to test.cmd.

Reply:
How about `test.args -v`?

Reply:
I removed -v anyway. That flag is not useful other than debugging.